openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -142,12 +142,22 @@
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
+        In case writing an audit record fails, all the API servers increase the
+        "apiserver_audit_error_total" metric. A PrometheusRule needs to be
+        created to create an alert in case the "apiserver_audit_error_total"
+        metric increases.
 
         https://issues.redhat.com/browse/CMP-161
```

Comment:
is the link still needed? Could we close this as it's possible and just handle it in the Compliance Operator?

Reply:
Thanks, this was a leftover

Reply:
the ticket URL in the commit message and the PR text was also fixed